app/layout.tsx

```diff
@@ -1,21 +1,20 @@
-import Footer from "@/components/Footer/Footer";
-import GitHubCorner from "@/components/GitHubCorner/GitHubCorner";
-import MetaTags from "@/components/Layout/MetaTags";
-import Navbar from "@/components/Navbar/Navbar";
-import "@fortawesome/fontawesome-svg-core/styles.css";
-import { Analytics } from "@vercel/analytics/react";
-import styles from "./layout.module.css";
-import "/styles/_styles.css";
+import Footer from '@/components/Footer/Footer';
```

Comment:
@Jasminvillatoro Do you have a user setting in your code editor that applies single quotes instead of double quotes? A lot of the diff for this PR looks to be formatting updates so it might make sense to add a Preettier config since we don't have one yet!

Would you be comfortable adding that as part of this PR? 

https://prettier.io/docs/en/configuration.html

Reply:
Feel free to resolve this conversation